rosdep/python.yaml

```diff
@@ -3452,6 +3452,18 @@ python-scipy:
     yakkety_python3: [python3-scipy]
     zesty: [python-scipy]
     zesty_python3: [python3-scipy]
+python-scp:
+  debian: [python-scp]
+  fedora:
+    pip:
+      packages: [scp]
+  ubuntu:
+    xenial: [python-scp]
```

Comment:
Since these are all the same they don't need to be called out by specific version.